universe/ValueRefs.h

```diff
@@ -448,8 +448,8 @@ struct FO_COMMON_API Operation final : public ValueRef<T>
 
 /* Convert between names and MeterType. Names are scripting token, like Population
  * and not the MeterType string representations like METER_POPULATION */
-[[nodiscard]] FO_COMMON_API MeterType          NameToMeter(const std::string& name);
-[[nodiscard]] FO_COMMON_API const std::string& MeterToName(MeterType meter);
+[[nodiscard]] FO_COMMON_API MeterType          NameToMeter(const std::string_view name);
+[[nodiscard]] FO_COMMON_API const std::string_view MeterToName(const MeterType meter);
```

Comment:
```suggestion
[[nodiscard]] FO_COMMON_API std::string_view MeterToName(const MeterType meter);
```
`const` of by-value return type is unnecessary. maybe also could impair return value optimization, but I'm just guessing.

Reply:
I think this might've been the source of the initial compilation issues I ran into. This allowed me to use the `std::string` constructor with `string_view`values.